core/src/network/actions/custom_send.rs

```diff
@@ -0,0 +1,81 @@
+extern crate futures;
+use crate::{
+    action::{Action, ActionWrapper, DirectMessageData},
+    context::Context,
+    instance::dispatch_action,
+    network::direct_message::{CustomDirectMessage, DirectMessage},
+};
+use futures::{
+    future::Future,
+    task::{LocalWaker, Poll},
+};
+use holochain_core_types::{
+    cas::content::Address,
+    error::HolochainError,
+};
+use snowflake::ProcessUniqueId;
+use std::{
+    pin::{Pin, Unpin},
+    sync::Arc,
+    thread::sleep,
+    time::Duration,
+};
+
+/// SendDiretMessage Action Creator for custom (=app) messages
```

Comment:
```suggestion
/// SendDirectMessage Action Creator for custom (=app) messages
```

Reply:
Does the code below mean the timeout is 60 seconds? 
Can that also be added into this comment description if so?